### DISCUSSION TITLE
API endpoints

### DISCUSSION THREAD

**Discussion by happydogcode** [community/community#24908] [Apps, API and Webhooks]
<p>Where can I find out which API endpoints my GitHup App can hit? TIA!</p>

**Comment by lee-dohm** [comment, marked as answer]
<p><a href="https://developer.github.com/v3/apps/available-endpoints/" rel="nofollow noopener">https://developer.github.com/v3/apps/available-endpoints/</a> (and stay tuned to this page - we’re enabling more endpoints all the time.)</p>
